spec/views/spotlight/sir_trevor/blocks/_solr_documents_carousel_block.html.erb_spec.rb

```diff
@@ -8,10 +8,10 @@
 
   before do
     allow(block).to receive(:each_document).and_return([
-      [{}, SolrDocument.new(id: 1)],
-      [{}, SolrDocument.new(id: 2)],
-      [{}, SolrDocument.new(id: 3)]
-    ])
+                                                         [{}, SolrDocument.new(id: 1)],
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.